src/wkt/src/wrappers.rs

```diff
@@ -286,6 +332,31 @@ mod test {
         Ok(())
     }
 
+    #[test_case(f32::INFINITY as FloatValue, "Infinity", "FloatValue")]
+    #[test_case(f32::NEG_INFINITY as FloatValue, "-Infinity", "FloatValue")]
+    #[test_case(f32::NAN as FloatValue, "NaN", "FloatValue")]
+    #[test_case(9876.5 as FloatValue, 9876.5, "FloatValue")]
+    fn test_wrapper_float<V>(input: FloatValue, value: V, typename: &str) -> Result
```

Comment:
To reuse this function for DoubleValue, it looks like we could us the num::Float trait instead of FloatValue: https://docs.rs/num/latest/num/trait.Float.html. But we would need to add a dependency, I think. Is there a better way to do this?

Reply:
I think you can define your own trait in this file.  Argh, my computer is rebooting... got to go.

Reply:
Ack, will keep this in mind when expanding to doubles :)